api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/TxCurrCohortQueries.java

```diff
@@ -228,8 +211,9 @@ public CohortDefinition getTxCurrCompositionCohort(String cohortName, CohortDefi
 		TxCurrComposition.addParameter(new Parameter("onOrBefore", "onOrBefore", Date.class));
 		TxCurrComposition.addParameter(new Parameter("location", "location", Location.class));
 		TxCurrComposition.addParameter(new Parameter("effectiveDate", "effectiveDate", Date.class));
+		TxCurrComposition.addParameter(new Parameter("locations", "location", Location.class));
 		TxCurrComposition.getSearches().put("1", new Mapped<CohortDefinition>(inARTProgramAtEndDate,
-		        ParameterizableUtil.createParameterMappings("onOrBefore=${onOrBefore},location=${location}")));
+		        ParameterizableUtil.createParameterMappings("onOrBefore=${onOrBefore},locations=${location}")));
```

Comment:
`locations` should be singular since we are talking about just one location

Reply:
@nthfloor if we use `location` the code will not work. The parameter name must match the field on the Cohort class we are using.

Reply:
Then please update the Cohort class

Reply:
We need to try and unify the parameter names we are using - we have too many variations

Reply:
It's the framework class `InProgramCohortDefinition`, we can't change

Reply:
No that is just a label we defined: have a look at L153 in the `GenericCihortQueries` file - it was named as `locations` but could have been named anything else really

Reply:
Sorry Nathan, it's not. You will not have errors but the location will not be taken into account. Please test and see.